packages/bsp/common/lib/systemd/system/armbian-resize-filesystem.service

```
@@ -4,9 +4,10 @@
 
 [Unit]
 Description=Armbian filesystem resize
-Before=basic.target
+Before=basic.target first-boot-complete.target
 After=sysinit.target local-fs.target
 DefaultDependencies=no
+Wants=first-boot-complete.target
```

```suggestion
Before=basic.target
After=sysinit.target local-fs.target
DefaultDependencies=no
```

`armbian-resize-filesystem` may require a reboot to process. So we can't put it into `first-boot-complete.target`